src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx

```diff
@@ -2,3 +2,243 @@ export const meta = {
+import awsconfig from './aws-exports';
+Amplify.configure(awsconfig);
+```
+
+</Block>
+
+<Block name="Vue">
+
+The first step to using Amplify in the client is to install the necessary dependencies:
+
+<Block name="NPM">
+
+```
+npm install aws-amplify @aws-amplify/ui-vue
+```
+
+</Block>
+
+<Callout>
+
+If you are using Vue 2, please check out our legacy [documentation](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue). For Vite installs check out this [documentation](https://ui.docs.amplify.aws/vue/getting-started/installation#vite). 
+
+</Callout>
+
+The `@aws-amplify/ui-vue` package is a set of components that make it easy to integrate functionality like end-to-end authentication flows. 
+
+In your app's entry point, typically **index.js**, import and load the configuration file:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+import awsconfig from './aws-exports';
+Amplify.configure(awsconfig);
+```
+
+</Block>
+<Block name="Angular">
+
+Inside the app directory, install the Amplify Angular Library and run your app:
+
+```bash
+npm install --save aws-amplify @aws-amplify/ui-angular
+
+npm start
+```
+
+The `@aws-amplify/ui-angular` package is a set of Angular components and an Angular provider that helps integrate your application with the AWS Amplify Library.
+
+<Callout>
+
+**Angular CLI output warnings:** If you encounter optimization bailouts warnings using Angular 9+ due to CommonJS or AMD dependencies, you can use this [gist](https://gist.github.com/wlee221/6d98d96740bea6f53327b4db4a432616) to remove them. See the [Angular website](https://angular.io/guide/build#configuring-commonjs-dependencies) for more details.
+
+</Callout>
+
+#### Strictly typing `aws-exports`
+
+If you have TypeScript [strict mode](https://www.typescriptlang.org/tsconfig/#strict) on and see the error
+
+```
+Could not find a declaration file for module './aws-exports'. 'aws-exports.js' implicitly has an 'any' type.
+```
+
+Create a `aws-exports.d.ts` file on the same level as `aws-exports` with the following content:
+
+```ts
+declare const awsmobile: Record<string, any>
+export default awsmobile;
+```
+
+#### Importing the Amplify Angular UI Module
+
+Add the **Amplify Authenticator UI Module** to `src/app/app.module.ts`:
+
+```ts
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AmplifyAuthenticatorModule } from '@aws-amplify/ui-angular';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    /* configure App with AmplifyAuthenticatorModule */
+    AmplifyAuthenticatorModule
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+</Block>
+</BlockSwitcher>
+
+Well done! You now have your Amplify Libraries set up and configured.
+
```

Comment:
@renebrandel Comment "I think we should add one more step here to quickly show how customers can read/write to their database (but then let customers expand on it in the following pages). It closely follows the codegen section." 

Moved CodeGen down to review here with the added Mutate and Query data from current Getting Started Guide which is also below. This is not intended to be the final read but rather pulling all related CTAs to this area so easier to edit and review.

Reply:
collapse 223 - 230 into  184.